ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```
@@ -620,6 +620,20 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-hypershift-guest-longduration-f14
+  cron: 10 9 14,28 * *
```

```suggestion
  cron: 13 18 4,18 * *
```
The cron should generate by the `./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh`
```console
./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-ipi-ovn-hypershift-guest-longduration-f14 openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml 
cron: 13 18 4,18 * *
```

---

Okay, will update.